app/src/main/java/com/vinh/dictionary_1/utis/binding/BindingUtils.java

```diff
@@ -102,6 +104,12 @@ public static void setWebViewContent(WebView webView, String content) {
 
     @BindingAdapter({ "webViewClient" })
     public static void setWebViewClient(WebView webView, WebViewClient webViewClient) {
+        if (Build.VERSION.SDK_INT >= 19) {
+            webView.setLayerType(View.LAYER_TYPE_HARDWARE, null);
+        }
```

Comment:
ERROR: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).